chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/curesStatistics/email/spreadsheet/UscdiCriteriaByAcbWorksheet.java

```diff
@@ -44,13 +44,18 @@ public Workbook populate(Workbook workbook) {
 
         Integer currentRowIdx = 1;
         Row currentRow;
+        String currentCriteria = "";
```

Comment:
Should this be `previousCriteria`? In the if you're checking if the current criteria = the stat.getcriteria, and that seems like it should always be true...